lib/active_model/serializer.rb

```diff
@@ -476,9 +479,7 @@ def scope
 
     def _serializable_hash
       return nil if @object.nil?
-      @node = attributes
```

Comment:
yeah, they do about 50% of the time. Something to do with the way fake objects are generated for testing.